lib/antlr-gemerator.rb

```diff
@@ -28,7 +28,7 @@ def create(options)
       bindings = {
         gem_name:            generator.gem_name,
         ext_name:            generator.ext_name,
-        gem_namespace:       generator.parser_ns,
+        gem_namespace:       generator.capitalize(generator.parser_ns),
```

Comment:
gem_namespace is only used in templates/version.rb.erb - module names in ruby are capitalized because they are constants
module <%= gem_namespace %>

and also uses in templates/gemspec.erb, and this refers to the version abouve which would be capitalized

s.version = ::<%- gem_namespace %>::VERSION

Reply:
Hmm interesting, so I guess `generator.parser_ns` isn't always capitalized? I must've just gotten lucky so far 😅